packages/jupyterlab-gridstack/package.json

```diff
@@ -43,8 +43,8 @@
     "watch:src": "tsc -w"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^4.0.0",
-    "@jupyter-widgets/jupyterlab-manager": "^3.0.0",
+    "@jupyter-widgets/base": "^4.0.0-alpha.2",
```

Comment:
Was this change intentional? Looks like it reverts some parts of https://github.com/voila-dashboards/voila-gridstack/pull/100

Reply:
Good catch - I started over from reverting a commit a miss that one. I open #112 to redo the changes.